src/seml/start.py

```diff
@@ -199,8 +199,8 @@ def start_sbatch_job(collection, exp_array, unobserved=False, name=None,
                      and exp_array[0][0]['seml']['conda_environment'] is not None)
 
     # Construct Slurm script
-    template = pkg_resources.resource_string(__name__, "slurm_template.sh").decode("utf-8")
-    prepare_experiment_script = pkg_resources.resource_string(__name__, "prepare_experiment.py").decode("utf-8")
+    template = importlib.resources.files(__name__).joinpath("slurm_template.sh").read_bytes().decode("utf-8")
+    prepare_experiment_script = importlib.resources.files(__name__).joinpath("prepare_experiment.py").read_bytes().decode("utf-8")
```

Comment:
Starting an experiment via `seml col add xyz.yaml start` fails with `TypeError: 'seml.start' is not a package`.

Reply:
Should be fixed.